go/vt/sqlparser/parse_test.go

```diff
@@ -1648,11 +1648,11 @@ var (
 			input:  "optimize foo",
 			output: "otheradmin",
 		}, {
-			input:  "lock tables foo",
-			output: "otheradmin",
+			input:  "lock tables foo read",
```

Comment:
Should have a test for many tables

Reply:
Many different kinds of locks

Reply:
Actually just move these tests out of here since you have a dedicated method

Or move them up here, take your pick